rpc/dataconn/timeoutconn/timeoutconn_iovec_int8.go

```diff
@@ -0,0 +1,17 @@
+// +build illumos solaris
+
+package timeoutconn
+
+import (
+	"syscall"
+	"unsafe"
+)
+
+func makeIovec(byteVec []byte) syscall.Iovec {
+	v := syscall.Iovec{
+		Base: (*int8)(unsafe.Pointer(&byteVec[0])),
```

Comment:
need to read up whether this is really safe, but don't know why not
`byte` is a type alias for `uint8`, and casting the `uint8` to `int8` _should_ be safe.
https://golang.org/pkg/builtin/#byte